### PR TITLE
Add markers from server

### PR DESCRIPTION
**Pull request by thompsond**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/AddMarkers.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import java.util.List;
+import java.util.ArrayList;
+import com.google.gson.Gson;
+
+@WebServlet("/add-markers")
+public class AddMarkers extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      List<Marker> markers = new ArrayList<>();
+      markers.add(new Marker(33.969036, -84.538326, "Tip Top Donuts"));
```
Comment (theannielin) [inline portfolio/src/main/java/com/google/sps/servlets/AddMarkers.java:31]: Do you always want to have these two specific markers?

Comment (thompsond) [reply portfolio/src/main/java/com/google/sps/servlets/AddMarkers.java:31]: I think so, the walkthrough just suggested that some of the markers should come from the server.

Comment (theannielin) [reply portfolio/src/main/java/com/google/sps/servlets/AddMarkers.java:31]: I see, seems fine then if it's intentional.

**Review by theannielin — APPROVED** [review]
LGTM. Nice class!

**Pull request merged**